modules/core/client/directives/show-errors.client.directives.js

```diff
@@ -1,79 +1,87 @@
-'use strict';
-
-/**
- * Edits by Ryan Hutchison
- * Credit: https://github.com/paulyoder/angular-bootstrap-show-errors */
-
-angular.module('core')
-  .directive('showErrors', ['$timeout', '$interpolate', function ($timeout, $interpolate) {
-    var linkFn = function (scope, el, attrs, formCtrl) {
-      var inputEl,
-        inputName,
-        inputNgEl,
-        options,
-        showSuccess,
-        toggleClasses,
-        initCheck = false,
-        showValidationMessages = false,
-        blurred = false;
-
-      options = scope.$eval(attrs.showErrors) || {};
-      showSuccess = options.showSuccess || false;
-      inputEl = el[0].querySelector('.form-control[name]') || el[0].querySelector('[name]');
-      inputNgEl = angular.element(inputEl);
-      inputName = $interpolate(inputNgEl.attr('name') || '')(scope);
-
-      if (!inputName) {
-        throw new Error('show-errors element has no child input elements with a \'name\' attribute class');
+(function () {
+  'use strict';
+
+  /**
+   * Edits by Ryan Hutchison
+   * Credit: https://github.com/paulyoder/angular-bootstrap-show-errors */
```

Comment:
Just leave the HTTP link in a single line comment and the multiline is unnecessary